api/nakadi-event-bus-api.yaml

```diff
@@ -888,6 +888,7 @@ definitions:
               Timestamp of creation of the Event generated by the producer.
             type: string
             format: date-time
+            example: '1996-12-19T16:39:57-08:00'
```

Comment:
You are right. Date and time formatting is no simple task. Thanks for the reference. :+1: 

I checked JODA time and they use a slightly different format http://joda-time.sourceforge.net/apidocs/org/joda/time/format/ISODateTimeFormat.html#dateTime()  and http://joda-time.sourceforge.net/apidocs/org/joda/time/format/ISODateTimeFormat.html#dateTimeNoMillis()

The main difference is that they would format UTC as Z instead of +00:00.

But lets use Joda, so we do not have to be aware of the format :+1: